src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Persistence/Services/AssignmentService.cs

```diff
@@ -29,6 +30,136 @@ IDelegationRepository delegationRepository
     private readonly IRolePackageRepository rolePackageRepository = rolePackageRepository;
     private readonly IEntityRepository entityRepository = entityRepository;
     private readonly IDelegationRepository delegationRepository = delegationRepository;
+    private static readonly string RETTIGHETSHAVER = "rettighetshaver";
+    private static readonly Guid PartyTypeOrganizationUuid = new Guid("8c216e2f-afdd-4234-9ba2-691c727bb33d");
+
+    /// <inheritdoc/>
+    public async Task<IEnumerable<ClientDto>> GetClients(Guid toId, string[] roles, string[] packages, CancellationToken cancellationToken = default)
+    {
+        // Setup filter to be used for fetching client assignments
+        var clientFilter = assignmentRepository.CreateFilterBuilder();
```

Comment:
kan dette filteret flyttes ned på linje 70?

Reply:
Jupp, har flyttet litt rundt på kallene her noen ganger (gjorde også to kall før som jeg nå har fått samlet i _ett kall)._ Enig at det bør ligge samlet ved selve repo-kallet nå.